app/uk/gov/hmrc/gvmshaulierstatus/repositories/HaulierStatusRepository.scala

```diff
@@ -68,9 +68,8 @@ class HaulierStatusRepository @Inject() (
         .toFuture()
     )
 
-  def findAndUpdate(correlationId: CorrelationId, status: Status)(implicit
-    instant: Instant = Instant.now(Clock.systemUTC())
-  ): Future[Option[String]] =
+  def findAndUpdate(correlationId: CorrelationId, status: Status): Future[Option[String]] = {
```

Comment:
If we're removing this parameter in both these defs just to get rid of a wartremover `DefaultArguments` warning, I think we're best off suppressing it instead. Looks like the parameter has been added to allow unit testing to confirm the respective date/time field is being updated as expected.

Reply:
...and if we do want to decide to get rid of the parameter, we presumably don't need to make the val implicit and probably don't need a val at all. 

Think I'd be in favour of suppressing this particular warning though and keeping the test.

Reply:
done